doc/source/user_guide/index.rst

```diff
@@ -82,8 +82,8 @@ This code uses the Sherlock Part Library to update the parts list:
             "Tutorial",
             "Main Board",
             "Sherlock Part Library",
-            "Both",
-            "Error"
+            UpdatesPartsListRequestMatchingMode.BOTH,
+            UpdatesPartsListRequestDuplicationMode.ERROR

```

Comment:
This class name was changed. Originally it was only used for UpdatePartsListRequest but got changed to be used for multiple requests.
These should be changed to `PartsListSerchMatchingMode`.

Reply:
Done